### PR TITLE
fix: #7 and add `-i` flag to sed cmd

### LINKED ISSUE #7: Makefile does not respect DESTDIR
Hi, I'm the AUR maintainer for this package. I believe @MCotocel had asked earlier if I could start building this with the Makefile instead of directly invoking `go build`, however, the Makefile in question doesn't seem to respect whatever DESTDIR is set to. This is important for Arch Linux, as packages are not allowed to directly install to `/usr/bin` without using `pacman`.

This is what I get from an unreleased hilbish PKGBUILD, using the latest release:
```shell
make: Warning: File 'Makefile' has modification time 5828 s in the future
'hilbish' -> '/usr/bin/hilbish'
install: cannot create regular file '/usr/bin/hilbish': Permission denied
make: *** [Makefile:9: install] Error 1
A failure occurred in package().
Aborting...
```

And this is what I have in the PKGBUILD:
```shell
package() {
  cd "$pkgname-$pkgver"
  DESTDIR="$pkgdir/" make install
```

`DESTDIR` is set to `$pkgdir`, which expands to the package directory. So ideally it should install to `$pkgdir/usr/bin/hilbish`

### PR DESCRIPTION
**Pull request by JavaCafe01**
pls test before merge :cry:

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,22 +1,23 @@
...
 	@echo "Hilbish Installed"
 
 uninstall:
 	@rm -vrf \
-			"$(BINDIR)/hilbish" \
-			"$(LIBDIR)"
-	@sed '/\/usr\/bin\/hilbish/d' /etc/shells
+			"$(DESTDIR)$(BINDIR)/hilbish" \
+			"$(DESTDIR)$(LIBDIR)"
+	@sed -i '/$(DESTDIR)\/usr\/bin\/hilbish/d' /etc/shells
```
Comment (devins2518) [inline Makefile:20]: We can probably simplify this to `@sed -i '/$(DESTDIR)$(BINDIR)\/hilbish/d' /etc/shells`

Comment (JavaCafe01) [reply Makefile:20]: True

File: Makefile [hunk 2]
```diff
@@ -1,22 +1,23 @@
...
 
 build:
 	@go build
 
-install:
-	@install -v -d "$(BINDIR)/" && install -m 0755 -v hilbish "$(BINDIR)/hilbish"
-	@mkdir -p "$(LIBDIR)"
-	@cp libs preload.lua .hilbishrc.lua "$(LIBDIR)" -r
-	@echo /usr/bin/hilbish >> /etc/shells
+install: build
```
Comment (TorchedSammy) [inline Makefile:9]: whyd u add back build >:|

Comment (JavaCafe01) [reply Makefile:9]: some people just do make install, and if they do it wont install cuz its not built :/ this prob is only on ur system lol

File: Makefile (hunk 1 above)
Comment (JavaCafe01) [reply Makefile:20]: :(

Comment (devins2518) [reply Makefile:20]: Actually `@sed -i '/hilbish/d' /etc/shells` will work fine

File: Makefile (hunk 2 above)
Comment (TorchedSammy) [reply Makefile:9]: `sudo make` is still available tho

**Comment by JavaCafe01** [comment]
@TorchedSammy ready for merge ig

**Review by TorchedSammy — APPROVED** [review]
seems good

**Pull request merged**